tests/CssToInlineStylesTest.php

```diff
@@ -267,6 +267,54 @@ public function testSelfClosingTags()
         $this->assertCorrectConversion($expected, $html, $css);
     }
 
+    public function testConversionAsciiRegular()
+    {
+        $html = '~';
+        $css = '';
+        $expected = "<p>{$html}</p>";
+        $this->assertCorrectConversion($expected, $html, $css);
+    }
+
+    public function testConversionAsciiDelete()
+    {
+        $html = '\u{007F}';
+        $css = '';
+        $expected = "<p>{$html}</p>";
+        $this->assertCorrectConversion($expected, $html, $css);
+    }
+
+    public function testConversionLowestCodepoint()
+    {
+        $html = '\u{0080}';
+        $css = '';
+        $expected = "<p>{$html}</p>";
+        $this->assertCorrectConversion($expected, $html, $css);
+    }
+
+    public function testConversionHighestCodepoint()
+    {
+        $html = '\u{10FFFF}';
+        $css = '';
+        $expected = "<p>{$html}</p>";
+        $this->assertCorrectConversion($expected, $html, $css);
+    }
+
+    public function testConversionIllegalCodepoint()
+    {
+        $html = '\u{110000}';
```

Comment:
shouldn't this use a double-quoted string if you intent to use U+110000 ? Your current code does not use an escape sequence here. It creates a pure-ASCII HTML

Reply:
🤦‍♂️ you're right, will fix!